sources/packages/backend/libs/services/src/notifications/notification/notification-actions.service.ts

```diff
@@ -150,6 +151,42 @@ export class NotificationActionsService {
     );
   }
 
+  /**
+   * Creates a notification when an Offering Change Request is inprogress with the student.
+   * @param notification input parameters to generate the notification.
+   * @param auditUserId user that should be considered the one that is causing the changes.
+   * @param entityManager optional entity manager to execute in transaction.
+   */
+  async saveOfferingChangeRequestInProgressWithStudent(
```

Comment:
`saveInstitutionRequestChangeNotification` ?

Reply:
If there is a common template for all 3 scenarios, I will update the names everywhere.